src/components/ModalWindowCheckout/ModalWindowCheckout.tsx

```diff
@@ -0,0 +1,111 @@
+import { FC, useState } from 'react';
+import styles from './styles.module.scss';
+import { PrimaryButton } from '../PrimaryButton';
+import { PrimaryButtons } from '@/enums/PrimaryButtons';
+import { useRemoveFromCartButton } from '@/hooks';
+import { useSelector } from 'react-redux';
+import { RootState } from '@/app/store';
+import { getCartProducts, getTotalCost } from '@/utils';
+import { useNavigate } from 'react-router-dom';
+import { Paths } from '@/enums';
+import { ModalWindowSuccess } from '../ModalWindowSuccess/ModalWindowSuccess';
```

Comment:
A little strange order of imports, it is better to do it in a more structured way. For example: external libraries -> internal libraries -> components